app/models/manager_refresh/inventory/persister.rb

```diff
@@ -155,12 +155,13 @@ def add_remaining_inventory_collections(defaults, options = {})
 
   def to_raw_data
     collections_data = collections.map do |key, collection|
-      next if collection.data.blank?
+      next if collection.data.blank? && collection.manager_uuids.blank? && collection.all_manager_uuids.nil?
```

Comment:
Why `.blank?` for `manager_uuids` but `.nil?` for `all_manager_uuids?`?  They both look like arrays...